app/styles/components/jace-blob.less

```diff
@@ -0,0 +1,153 @@
+.jace-blob-container {
+  position: relative;
+  width: var(--spacing-px-30);
+  height: var(--spacing-px-30);
+  border-radius: 100%;
+  transition: all 1s ease-in-out;
+
+  &::after,
+  &::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: var(--spacing-px-30);
+    height: var(--spacing-px-30);
+    border-radius: 50%;
+    transform: translate(-50%, -50%) scale(0);
+    pointer-events: none;
+    z-index: 0;
+    border: 0.5px solid #ed21ff;
+    animation: none;
+    transition: all 1s ease-in-out;
+    border-width: 0;
+    box-shadow: none;
+    background: none;
+    transform: translate(-50%, -50%) scale(0);
+    opacity: 0;
+  }
+
+  .jace-blob {
+    border-style: solid;
+    border-color: transparent;
+    border-width: 0;
+    background: transparent;
+    border-radius: 50%;
+    transition: all 1s ease-in-out;
+    box-shadow: none;
+    animation: none;
+  }
+}
+
+.jace-blob-container--active {
+  &::after,
+  &::before {
+    transform: translate(-50%, -50%) scale(1);
+    border: 0.5px solid #ed21ff;
+    animation: ripple-border 5s ease-in-out infinite;
+    transition: all 1s ease-in-out;
+    pointer-events: none;
+    z-index: 0;
+    opacity: 1;
+  }
+
+  &::before {
+    animation-delay: 2.5s;
+  }
+
+  .jace-blob {
+    border-style: solid;
+    border-color: transparent;
+    border-width: 0.5px;
+    background: linear-gradient(white, white) padding-box, var(--color-smart-gradient-4000) border-box;
+    border-radius: 50%;
+    transition: all 1s ease-in-out;
+    animation: gradient-fade 5s ease-in-out infinite;
+    box-shadow: 2.29px 2.29px 4.58px 0px rgba(255, 255, 255, 1) inset, 0px 0px 10px 0px rgba(83, 94, 252, 0.6);
+  }
+}
+
+.jace-blob {
+  position: relative;
+  width: var(--spacing-px-30);
+  height: var(--spacing-px-30);
+  overflow: hidden;
+  border-radius: 50%;
+  z-index: 1;
+  transition: all 1s ease-in-out;
+
+  img {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 200%;
+    height: 200%;
+    transform: translate(-50%, -50%);
+    object-fit: cover;
+    object-position: center;
+    transition: all 1s ease-in-out;
+
+    &.fade-in {
+      opacity: 1;
+    }
+    &.fade-out {
+      opacity: 0;
+    }
+  }
+}
+
+.jace-blob-container.fade-out {
+  transform: scale(0.9);
+  box-shadow: none;
+
+  &::after,
+  &::before {
+    border-width: 0;
+    animation: none;
+    transform: translate(-50%, -50%) scale(0);
+    opacity: 0;
+  }
+
+  .jace-blob {
+    box-shadow: none;
+    background: transparent;
+    border-width: 0;
+    transform: scale(0.9);
+    opacity: 0;
+  }
+}
+
+@keyframes ripple-border {
```

Comment:
nit: just so we prevent any future issues (possible naming overlaps), could we have `jace-blob-ripple-border` and `jace-blob-gradient-fade` as animation names ?